packages/account-v2/src/modules/src/index.d.ts

```diff
@@ -0,0 +1,36 @@
+import type { TSupportedDocuments } from '../../types';
+
+type TManualForm = {
+    document_expiry: string;
+    document_number: string;
+};
+
+type TManualDocumentTypes = 'driving_licence' | 'national_identity_card' | 'nimc_slip' | 'passport';
```

Comment:
are we still going to use snake case in this case?

Reply:
We are passing these values to API and have integrated this into our forms, need to look for a better way to handle this. It won't be part of this PR.